instrument.go

```diff
@@ -72,7 +72,7 @@ func (i *Instrumenter) instrument(name string, err error, elapsed time.Duration)
 
 func (i *Instrumenter) recover(name string) {
 	if r := recover(); r != nil {
-		log.WithField("rpc", name).Errorf("panic: %v\n%v", r, debug.Stack())
+		log.L().With(zap.String("rpc", name)).Sugar().Errorf("panic: %v\n%v", r, debug.Stack())
```

Comment:
@dim probably, `log.Error("panic", zap.String("rpc", name), zap.Any("panic", r), zap.String("stack", debug.Stack()))`?

Reply:
yeah, that one's a bit different, I wouldn't bother too much with that one